ftest.sh

```diff
@@ -243,9 +252,21 @@ args+=" $*"
 # shellcheck disable=SC2029
 # shellcheck disable=SC2086
 if ! ssh -A $SSH_KEY_ARGS ${REMOTE_ACCT:-jenkins}@"${nodes[0]}" "set -ex
-rm -rf $DAOS_BASE/install/tmp
-mkdir -p $DAOS_BASE/install/tmp
-cd $DAOS_BASE
+if $TEST_RPMS; then
+    rm -rf $PWD/install/tmp
+    mkdir -p $PWD/install/tmp
+    # set the shared dir
+    # TODO: remove the need for a shared dir by copying needed files to
+    #       the test nodes
+    export DAOS_TEST_SHARED_DIR=${DAOS_TEST_SHARED_DIR:-$PWD/install/tmp}
+    logs_prefix="/var/tmp"
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]